packages/@webex/plugin-meetings/src/constants.ts

```diff
@@ -375,6 +375,7 @@ export const EVENT_TRIGGERS = {
   MEETING_LOCUS_URL_UPDATE: 'meeting:locus:locusUrl:update',
   MEETING_STREAM_PUBLISH_STATE_CHANGED: 'meeting:streamPublishStateChanged',
 
+  MEETING_LLM_CONNECTED: 'meeting:llm:connected',
```

Comment:
agree with Arun here. we can change the name.

Reply:
Addressed this @sreenara